data/scripts/derby/q_test_extDB_init.derby.sql

```diff
@@ -0,0 +1,27 @@
+-- reference: https://medium.com/swlh/getting-started-with-apache-derby-9f6ca8dea873
+
+create table country (name char(30) not null unique);
+create table state (name char(30) not null unique,
+                    country_code int not null);
+create table city (name char(30) not null unique,
```

Comment:
create table if not exists table_name? I suggesting this because tables might not get dropped/created properly in a container, it might lead to flaky tests